src/test/java/nextstep/subway/line/LineAcceptanceTest.java

```diff
@@ -1,113 +1,72 @@
 import nextstep.subway.line.dto.LineResponse;
+import nextstep.subway.station.dto.StationResponse;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 import org.springframework.http.HttpStatus;
-import org.springframework.http.MediaType;
 
+import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 import java.util.stream.Collectors;
 
+import static nextstep.subway.line.LineTestFixture.BASE_LINE_URL;
+import static nextstep.subway.line.LineTestFixture.구분당선_요청_데이터;
+import static nextstep.subway.line.LineTestFixture.신분당선_요청_데이터;
+import static nextstep.subway.line.LineTestFixture.이호선_요청_데이터;
+import static nextstep.subway.line.LineTestFixture.지하철_노선_등록되어_있음;
+import static nextstep.subway.line.LineTestFixture.지하철_노선_목록_조회_요청;
+import static nextstep.subway.line.LineTestFixture.지하철_노선_생성_요청;
+import static nextstep.subway.line.LineTestFixture.지하철_노선_수정_요청;
+import static nextstep.subway.line.LineTestFixture.지하철_노선_제거_요청;
+import static nextstep.subway.line.LineTestFixture.특정_지하철_노선_조회_요청;
+import static nextstep.subway.station.StationTestFixture.강남역_요청_데이터;
+import static nextstep.subway.station.StationTestFixture.역삼역_요청_데이터;
+import static nextstep.subway.station.StationTestFixture.지하철역_등록되어_있음;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.junit.jupiter.api.Assertions.assertAll;
 
 @DisplayName("지하철 노선 관련 기능")
 class LineAcceptanceTest extends AcceptanceTest {
 
-    private final String BASE_LINE_URL = "/lines";
+    private void 지하철_노선_조회_데이터_확인(ExtractableResponse<Response> response, LineResponse... expectedLineResponses) {
+        List<LineResponse> lineResponses = new ArrayList<>(response.jsonPath()
+                .getList(".", LineResponse.class));
 
-    private LineRequest 신분당선_요청_데이터() {
-        return new LineRequest("신분당선", "bg-red-600");
+        노선_ID_확인(lineResponses, expectedLineResponses);
+        노선_포함_확인(lineResponses, expectedLineResponses);
     }
 
-    private LineRequest 구분당선_요청_데이터() {
-        return new LineRequest("구분당선", "bg-blue-600");
-    }
-
-    private LineRequest 이호선_요청_데이터() {
-        return new LineRequest("2호선", "bg-green-600");
-    }
-
-    private ExtractableResponse<Response> 지하철_노선_생성_요청(LineRequest lineRequest) {
-        return RestAssured.given()
-                .log()
-                .all()
-                .body(lineRequest)
-                .contentType(MediaType.APPLICATION_JSON_VALUE)
-                .when()
-                .post(BASE_LINE_URL)
-                .then()
-                .log()
-                .all()
-                .extract();
-    }
-
-    private ExtractableResponse<Response> 지하철_노선_목록_조회_요청() {
-        return 지하철_노선_목록_조회_요청(BASE_LINE_URL);
-    }
+    private void 노선_포함_확인(List<LineResponse> lineResponses, LineResponse[] expectedLineResponses) {
+        HashMap<Long, List<StationResponse>> stationResponses = new HashMap<>();
```

Comment:
```suggestion
        Map<Long, List<StationResponse>> stationResponses = new HashMap<>();
```

자바의 다형성을 활용하는게 좋겠네요.